src/components/List.js

```diff
@@ -1,35 +1,35 @@
 import React from 'react';
-import './main.css';
 import { FirestoreCollection } from 'react-firestore';
 import '../css/components/ItemsList.css';
 
-// export default function List() {
-//   return (
-//     <div className="List">
-//       <h1>The LIST view</h1>
-//     </div>
-//   );
-// }
 export default function List() {
   return (
     <div>
       <br />
       <br />
       <h1>LIST OF ITEMS</h1>
       <FirestoreCollection
-        path="items"
+        path="shoppingLists"
+        filter={['token', '==', 801]} // TODO => fetch the token from localSorage
         render={({ isLoading, data }) => {
           return isLoading ? (
             <div className="m-auto">Loading</div>
           ) : (
             <div>
-              <ul>
-                {data.map((item) => (
-                  <li key={item.id} className="list-item">
-                    <div className="name">{item.name}</div>
-                  </li>
-                ))}
-              </ul>
+              {
+                // just making sure we have a shoppingList before mapping through its items
+                data[0] ? (
```

Comment:
Hey! Sorry, I just saw your feedback. Didn't receive a notification. :( 
Yeah, I get what you mean. However, I think it might not be the best idea to create a new shopping list right after the token is generated. Because what if say, 100 users used our app (which is very unlikely to happen 😂) but no one actually had added any items to their shopping list yet that would create 100 empty shopping lists in the database. So whenever we'd search for a shopping list we'd have to go through all the lists in the database, even if they were empty, you know.

Reply:
I was wondering, what do you think if we redirect the user to the Home component and the button "create a new list" (this is assuming that the user doesn't have a token yet, right?). We could do this once we work together on merging both PRs?

Reply:
I think this week's issues are just about creating a token to authorize users and add that token with the items to the database. So maybe we just don't need to worry about if the user is redirected to an empty or doesn't exist shopping list yet.  😺

Reply:
@hajarNasr 

> Hey! Sorry, I just saw your feedback. Didn't receive a notification. :(
> Yeah, I get what you mean. However, I think it might not be the best idea to create a new shopping list right after the token is generated. Because what if say, 100 users used our app (which is very unlikely to happen 😂) but no one actually had added any items to their shopping list yet that would create 100 empty shopping lists in the database. So whenever we'd search for a shopping list we'd have to go through all the lists in the database, even if they were empty, you know.

That's a good point actually 👍

Reply:
You brought up some good points here. Do you want save a shopping list token to the database if there aren't any items added? Should a user be referred to an empty list? This isn't on the user story descriptions/AC for issue 3 and 4. I'm leaning towards Hajar's suggestion, but we can discuss this on Sunday.